csrankings.csv

```diff
@@ -6474,6 +6488,7 @@ Jordi Perello Muntan,Polytechnic University of Catalonia,http://people.ccaba.upc
 Jordi Torres,Polytechnic University of Catalonia,https://www.facebook.com/public/Jordi-Torres-Sanchez/school/Polytechnic-University-of-Catalonia-109077209117931,waODsw0AAAAJ
 Jorge Arturo Cobb,University of Texas at Dallas,http://www.utdallas.edu/~jcobb,NOSCHOLARPAGE
 Jorge C. Lucero,Universidade de Brasília,http://cic.unb.br/~lucero,YRGc9WwAAAAJ
+Jorge Cruz 0001,Universidade NOVA de Lisboa,https://userweb.fct.unl.pt/~jc,ENwGNokAAAAJ

```

Comment:
It says "no permission" to access this URL. I find that I cannot access many of the listed home pages.